section-05-production-model-package/regression_model/pipeline.py

```diff
@@ -0,0 +1,125 @@
+from feature_engine.encoding import (
+    RareLabelEncoder,
+    OrdinalEncoder,
+)
+
+# from feature-engine
+from feature_engine.imputation import (
+    AddMissingIndicator,
+    MeanMedianImputer,
+    CategoricalImputer,
+)
+from feature_engine.selection import DropFeatures
+from feature_engine.transformation import LogTransformer
+from feature_engine.wrappers import SklearnTransformerWrapper
+from sklearn.linear_model import Lasso
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import MinMaxScaler, Binarizer
+
+from regression_model.config.core import config
+from regression_model.processing import preprocessors as pp
```

Comment:
I thought we would call this features instead of preproessors

Reply:
renaming